net-mgmt/zabbix5-proxy/src/opnsense/scripts/OPNsense/Zabbixproxy/setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Setup database directory
+mkdir -p /var/db/zabbix
+chown -R zabbix:zabbix /var/db/zabbix
+chmod 755 /var/db/zabbix
+
+# Setup logging
+touch /var/log/zabbix_proxy.log
+chown -R zabbix:zabbix /var/log/zabbix_proxy.log
```

Comment:
Would you mind moving the logs to `/var/log/zabbix` to have them in the same folder as the agent logs?

Reply:
Sure, why not